packages/release/label-data-pref.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Label preferred data partition
+DefaultDependencies=no
+Conflicts=shutdown.target
+# This is the partition GUID for the 'BOTTLEROCKET-DATA-PREF' partition
+Wants=dev-disk-by\x2dpartuuid-5b94e8df\x2d28b8\x2d485c\x2d9d19\x2d362263b5944c.device
+After=dev-disk-by\x2dpartuuid-5b94e8df\x2d28b8\x2d485c\x2d9d19\x2d362263b5944c.device
+
+[Service]
+Type=oneshot
+
+# Resize the partition, whether or not it resides on the same disk as /.
+ExecStart=/usr/bin/systemd-repart --dry-run=no /dev/disk/by-partuuid/5b94e8df-28b8-485c-9d19-362263b5944c
```

Comment:
Since this grows the partition in addition to re-labelling the partition, the `systemd-repart` call in `repart-local.service` becomes redundant. I decided to keep `repart-local.service` as is because there's no reason to change it since `systemd-repart` is no-op if the partition is already set up.

Reply:
If someone brings their own `BOTTLEROCKET-DATA` partition, that would suppress the other oneshots, and we'd still want `systemd-repart` to run in `repart-local.service`.